### ISSUE TITLE
iperf3-head(0ea9cb624fb, 3.17+) crashes at the end (sigreturn), stock iperf (iperf 3.6 (cJSON 1.5.2)) doesn't

### ISSUE THREAD

**Issue by thehajime** [thehajime/linux#5]
sigreturn related ?

**Comment by thehajime** [comment]
hmm, built on alpine 3.20.3, it won't reproduce the issue...

```diff
diff --git a/configure b/configure
index ab56785..c7e5d0e 100755
--- a/configure
+++ b/configure
@@ -2852,7 +2852,7 @@ fi
 
 if test "x$enable_static_bin" = xyes
 then :
-  LDFLAGS="$LDFLAGS --static"
+  LDFLAGS="$LDFLAGS --static-pie"
 
 fi
 
```

with `env LDFLAGS=-pie ./configure --enable-static-bin` won't generate a pie file.

**Comment by thehajime** [comment]
the binary which I can reproduce, is built with musl 1.1.22, which seems to be included in alpine 3.10.2.
(I don't remember how I built with crashed iperf3 with git master => looks like built w/ 3.10.2).

If we built with the newer musl, looks like should be okay.  invalid ?
